definitions/apm-application/relationships.yml

```diff
@@ -0,0 +1,30 @@
+relationships:
+  # Select entityGuid, peer.hostname from Span where peer.hostname is not null and category =
+  # 'datastore'
+  # ExtService in this case is a DB
+  - provider: newrelic
+    eventType: Span
+    rootEntityNodeType: SOURCE
+    relationship: CALLS
+    relatedEntity:
+      domain: EXT
+      type: SERVICE
+      domainId: peer.hostname
+      encodeIdentifierInGUID: true
+    conditions:
```

Comment:
multi conditions OR | AND

Reply:
Do we need them now?

Reply:
I thought we agreed to add and|or later?

Reply:
PRs > VC for this reason!

My preferences would be only one attribute. And, if we do have a real use case for N attributes, then only suppor AND acrosss all of them. Or is implemented by creating a copy of the rule  (this is what we do in entity synthesis, to avoid complicating the syntax).

I can see a case for going to N attributes with AND because that way we could just not have `instrumentation.provider`. 

But I'm reluctant to going beyond that because logic operators to the yaml it feels antipattern-y to me (e.g. yaml stops being configuration and starts begin a programming language).

Reply:
I agree with starting with support for just AND. However, multiple AND clauses would imply more filtering thus less processing so I'm all for permitting multiple off the bat. what you think

Reply:
We didn't want to allow the `AND` on entity definitions, but here it should be fine right?

Reply:
Yes, N match conditions that are all required should not be problematic.